README.rst

```diff
@@ -1 +1,46 @@
-# trytond-quickbooks-payroll
+Trytond QuickBooks Payroll
+==========================
+
+Module helps to import payroll entries from quickbooks to tryton. It expects a
+csv file in following format::
+
+  Date,Num,Type,Source Name,Payroll Item,Wage Base,Amount
+  04/01/2015,309333,Cash,"Blow, Joe",Salary Expense,,-100000
+  ,,,"Blow, Joe",Federal Income Taxes Payable,,15000
+  ,,,"Blow, Joe",State Income Taxes Payable,,5000
+  ,,,"Blow, Joe",FICA Taxes Payable,,7650
+  ,,,,,,72350
+
+How to use it?
+--------------
+
+First you will have to map tryton employee with quickbooks source name.
+Go to ``Party >> Configuration >> Companies >> Employees``
+
+.. image:: images/employee.png
+
+Map accounts with quickbooks payroll item,
+``Financial >> Configuration >> Quickbooks Payroll``
+
+.. image:: images/payroll.png
+
+Now go to ``Import Payroll Items wizard``, choose Credit Account,
```

Comment:
Now go to ``Import Payroll Items wizard`` (Financial >> Entries >> Import Payroll Items)